server/status.go

```diff
@@ -52,10 +51,13 @@ const (
 	// statusLocalStacksKey exposes stack traces of running goroutines.
 	statusLocalStacksKey = statusLocalKeyPrefix + "stacks"
 
-	// statusNodesKeyPrefix exposes status for each of the nodes the cluster.
-	// GETing statusNodesKeyPrefix will list all nodes.
-	// Individual node status can be queried at statusNodesKeyPrefix/NodeID.
-	statusNodesKeyPrefix = statusKeyPrefix + "nodes/"
+	// statusNodesSingleKeyPrefix and statusNodeAllKeyPrefix exposes status for
+	// each of the nodes the cluster.
+	// statusNodesKeyPrefix/nodes -> lists all nodes
+	// statusNodesKeyPrefix/nodes/ -> lists all nodes
+	// statusNodesKeyPrefix/nodes/{NodeID} -> shows only the status for that single node
+	statusNodesSingleKeyPrefix = statusKeyPrefix + "nodes/"
+	statusNodesAllKeyPrefix    = statusKeyPrefix + "nodes"
```

Comment:
There are no wildcards, so I can't put in: .../nodes/{NodeID}
So it will hit the largest one that it sees.  Which is why /nodes/ will also go to handleNodeStatus instead of handleNode<b>s</b>Status, but when no ID is parsed, it will call handleNodeStatus.